variables.tf

```diff
@@ -8,3 +8,14 @@ variable "region" {
   type    = string
   default = "eu-west-2"
 }
+
```

Comment:
Remove these if not required rather than leave as comments

Reply:
done